lab-davidw/test/client-test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const expect = require('chai').expect;
+const testClient = require('../model/client.js');
+
+describe('client test', () => {
+  before('Create Client Object', () => {
+    this.client = new testClient();
+  });
+  it('should return an object', done => {
+    expect(this.client).to.be.an.instanceof(Object);
+    done();
+  });
+  it('should contain socket', done => {
```

Comment:
This test isn't really valid because you're explicitly setting your socket to Port 3000 already in Line 15. If you wanted to test for a matching socket number, I'd also require in your server module to listen on Port 3000 and just run your server.